core/src/navigation/LeftNav.html

```diff
@@ -475,8 +475,13 @@ <h5 class="lui-flyout-sublist__title fd-has-type-minus-1 fd-has-color-text-4">{$
   }
 
   function setBurgerTooltip() {
-    if (document.body.classList.contains('lui-semiCollapsible')) {
-      burgerTooltip = document.body.classList.contains('semiCollapsed') ? NavigationHelpers.getBurgerTooltipConfig()[0] : NavigationHelpers.getBurgerTooltipConfig()[1];
+    if (NavigationHelpers.getBurgerTooltipConfig()) {
+      const [collapseNavTooltip, expandNavTooltip] = NavigationHelpers.getBurgerTooltipConfig();
+      if (collapseNavTooltip && expandNavTooltip) {
+        if (document.body.classList.contains('lui-semiCollapsible')) {
+          burgerTooltip = document.body.classList.contains('semiCollapsed') ? collapseNavTooltip : expandNavTooltip;
+        }
+      }
```

Comment:
```suggestion
    if (!NavigationHelpers.getBurgerTooltipConfig()) {
        return;
    }

    const [collapseNavTooltip, expandNavTooltip] = NavigationHelpers.getBurgerTooltipConfig();
    const hasSemiCollapsible = document.body.classList.contains('lui-semiCollapsible');
    if (collapseNavTooltip && expandNavTooltip && hasSemiCollapsible) {
        burgerTooltip = document.body.classList.contains('semiCollapsed') ? collapseNavTooltip : expandNavTooltip;
     }
```

Reply:
Something like above which improves the readability. Please test if it works and feel free to name the parameter better.